zero_bin/common/src/prover_state/persistence.rs

```diff
@@ -20,7 +23,20 @@ use super::{
 const CIRCUITS_DIR: &str = "circuits/";
 const PROVER_STATE_FILE_PREFIX: &str = "prover_state";
 const VERIFIER_STATE_FILE_PREFIX: &str = "verifier_state";
-const CARGO_WORKSPACE_DIR_ENV: &str = "CARGO_WORKSPACE_DIR";
+const ZK_EVM_WORKSPACE_DIR_ENV: &str = "ZK_EVM_WORKSPACE_DIR";
+
+/// We version serialized circuits by the kernel hash they were serialized with,
+/// but we really only need a few of the starting hex nibbles to reliably
+/// differentiate.
+const NUM_HASH_NIBS_TO_USE_IN_CIRCUIT_VERSION: usize = 8;
```

Comment:
I find the name overly long 🙈 
Maybe `KERNEL_HASH_PREFIX` would carry the proper meaning while being more consistent ?

Reply:
Sure lets swap that out.